### PR TITLE
Updated dependencies to latest versions for all benchmark tests 

### PR DESCRIPTION
**Pull request by harshita19244**

### CODE REVIEW

File: opentracing-benchmark-java-jaxrs/pom.xml [hunk 1]
```diff
@@ -33,21 +33,21 @@
 		<!--
             JMH version to use with this project.
           -->
-		<jmh.version>1.21</jmh.version>
+		<jmh.version>1.31</jmh.version>
```
Comment (gsoria) [inline opentracing-benchmark-java-jaxrs/pom.xml:36]: The version `1.32` was released yesterday :)

File: opentracing-benchmark-java-jaxrs/pom.xml [hunk 2]
```diff
@@ -70,9 +70,14 @@
 		<dependency>
 			<groupId>org.mybatis.spring.boot</groupId>
 			<artifactId>mybatis-spring-boot-starter</artifactId>
-			<version>1.3.2</version>
+			<version>2.1.4</version>
 		</dependency>
-
+        <dependency>
```
Comment (gsoria) [inline opentracing-benchmark-java-jaxrs/pom.xml:75]: Some weird indent'n here.

File: opentracing-benchmark-java-jdbc/pom.xml [hunk 3]
```diff
@@ -69,7 +69,13 @@
 		<dependency>
 			<groupId>org.mybatis.spring.boot</groupId>
 			<artifactId>mybatis-spring-boot-starter</artifactId>
-			<version>1.3.2</version>
+			<version>2.1.4</version>
```
Comment (gsoria) [inline opentracing-benchmark-java-jdbc/pom.xml:72]: The version 2.2.0 is already available.

File: opentracing-benchmark-java-servlet-filter/pom.xml [hunk 4]
```diff
@@ -17,12 +17,12 @@
...
-        <haystack.version>0.2.5</haystack.version>
-        <jmh.version>1.21</jmh.version>
-        <jaeger.version>0.31.0</jaeger.version>
-        <undertow.version>2.0.21.Final</undertow.version>
+        <opentracing.version>0.33.0</opentracing.version>
+        <opentracing.java.servlet.filter.version>0.4.1</opentracing.java.servlet.filter.version>
+        <haystack.version>0.4.0</haystack.version>
+        <jmh.version>1.31</jmh.version>
+        <jaeger.version>1.6.0</jaeger.version>
+        <undertow.version>2.2.7.Final</undertow.version>
```
Comment (gsoria) [inline opentracing-benchmark-java-servlet-filter/pom.xml:25]: you could use the version `2.2.8.Final`

File: opentracing-benchmark-java-jaxrs/pom.xml [hunk 5]
```diff
@@ -33,21 +33,21 @@
...
-		<opentracing.version>0.31.0</opentracing.version>
-		<opentracing.jaxrs2.discovery.version>0.4.1</opentracing.jaxrs2.discovery.version>
+		<opentracing.version>0.33.0</opentracing.version>
+		<opentracing.jaxrs2.discovery.version>1.0.0</opentracing.jaxrs2.discovery.version>
 		<!--
             Jaeger version
           -->
-		<jaeger.version>0.31.0</jaeger.version>
-		<haystack.version>0.2.5</haystack.version>
+		<jaeger.version>1.6.0</jaeger.version>
```
Comment (gsoria) [inline opentracing-benchmark-java-jaxrs/pom.xml:49]: wow, this was a big jump.

File: opentracing-benchmark-spring-boot/pom.xml [hunk 6]
```diff
@@ -20,22 +20,28 @@
...
 		<!--
             Jaeger version
           -->
-		<jaeger.version>0.31.0</jaeger.version>
-		<haystack.version>0.2.5</haystack.version>
+		<jaeger.version>1.6.0</jaeger.version>
+		<haystack.version>0.4.0</haystack.version>
 	</properties>
 	<dependencies>
+		<dependency>
```
Comment (gsoria) [inline opentracing-benchmark-spring-boot/pom.xml:39]: Similar issue with indent'n here.

**Review by gsoria — COMMENTED** [review]
Thanks for these changes!, I put some initial comments/suggestions and I have two questions:

- I'm wondering if we should compile with newer versions of JDK (currently using 1.8)
- Are you planning to add the parent pom for dependency management as you suggested? I think it will remove a lot of duplicated definitions of dependencies and it will make it cleaner. :)

**Comment by harshita19244** [comment]
I added updates in the dependencies as pointed out by you, and now I am using a relatively new jdk version (14) for compilation.

**Comment by harshita19244** [comment]
@gsoria I made some further changes in the PR.  As of now, I have not defined a parent pom. I found out that new versions were being displayed in dependency management because the actual spring boot parent pom (that is defined as a parent ) is using older versions, although newer ones are available. Based on that, one possible fix is to override the required dependencies that our project is using by defining a dependency management tag. I wanted to define this as a common feature for the tests that were using the spring boot pom, but a pom file can only have a single parent. So, I have added the changes by defining a dependency management tag. Let me know of any possible improvements.

**Review by gsoria — APPROVED** [review]

**Comment by harshita19244** [comment]
@jpkrohling

File: opentracing-benchmark-java-jdbc/pom.xml [hunk 7]
```diff
@@ -12,10 +12,36 @@
 	<description>This project contains benchmark tests of opentracing java jax-rs.
 		The jax-rx example is based on: https://github.com/tachuela700/course-management
 	</description>
+	
+	<dependencyManagement>
```
Comment (jpkrohling) [inline opentracing-benchmark-java-jdbc/pom.xml:16]: Could you please reformat this file? The indentation looks odd.

**Review by jpkrohling — APPROVED** [review]
LGTM. I'll merge as is, but please reformat the file I pointed out in a follow-up PR.

**Pull request merged**